test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -3,13 +3,33 @@
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 
+import java.awt.*;
 import java.util.Arrays;
 import java.util.List;
 
 import org.junit.Test;
 
 public class UtilsTest {
 
+    @Test
+    public void testIsAnyNull() throws Exception {
+        // empty list
+        assertNoNull();
+
+        //has null
+        assertHasNull("qw", "e123", null);
+
```

Comment:
More test cases should be included. Eg, if you checking null, check different combinations such as null in the beginning, middle, or only null, etc.